collector/src/main/java/com/bitmovin/analytics/stateMachines/PlayerStateMachine.java

```diff
@@ -38,7 +38,7 @@ public PlayerStateMachine(BitmovinAnalyticsConfig config, BitmovinAnalytics anal
     public void enableHeartbeat() {
         heartbeatHandler.postDelayed(new Runnable() {
             public void run() {
-                long currentTimestamp = Util.getTimeStamp();
+                long currentTimestamp = Util.getElapsedTimestamp();
```

Comment:
Shouldn't the naming change a bit here? `timestamp` implies to me that it's an actual Time - instead it's a random timestamp that increases monotonically that can only be used for interval calculations. 
So maybe `intervalStart` or similar.

I'm a bit worried someone might do a `sample.time = currentTimestamp` or something else